omero/sysadmins/unix/install-web/web-deployment.txt

```diff
@@ -52,8 +52,8 @@ Virtualenv is an optional step:
    2016-08-24 16:20:14,252 [omego.artifa] INFO  Checking http://downloads.openmicroscopy.org/omero/5.2.5/./artifacts/OMERO.py-5.3.0-ice36-b29.zip
```

Comment:
for that I will see when we have the new walkthrough in so we can use litteralinclude